README.md

```diff
@@ -89,7 +89,7 @@ We really appreciate your volunteer spirit! You can contribute to the app in a f
 ## Available in many languages
 [![Crowdin](https://badges.crowdin.net/organized/localized.svg)](https://crowdin.com/project/organized)
 
-Currently, the app is fully or partially available in **20 languages**, including English, Spanish, Portuguese, French, German, Italian, Polish, Japanese, Russian, Ukrainian, Malagasy, and even more to come. If you'd like to contribute by translating Organized into the language of your heart, you're very welcome to do so!
+Currently, the app is fully or partially available in **20 languages**, including `English`, `Spanish`, `Portuguese`, `French`, `German`, `Italian`, `Polish`, `Japanese`, `Russian`, `Chineese`, `Ukrainian`, `Malagasy`, and even more to come. If you'd like to contribute by translating Organized into the language of your heart, you're very welcome to do so!
```

Comment:
@ux-git: just one small comment here: should it be `Chinese` instead of `Chineese`? Thanks.